pkgs/top-level/all-packages.nix

```diff
@@ -39592,6 +39592,12 @@ with pkgs;
     branch = "canary";
   };
 
+  discord-development = import ../applications/networking/instant-messengers/discord {
+    inherit lib stdenv;
+    inherit (pkgs) callPackage fetchurl;
+    branch = "development";
+  };
```

Comment:
```suggestion
  discord-development = callPackage ../applications/networking/instant-messengers/discord {
    branch = "development";
  };
```

Reply:
Changing it to a callPackage would break overriding acording to https://github.com/NixOS/nixpkgs/pull/179212#discussion_r914086385

Reply:
https://github.com/NixOS/nixpkgs/pull/222950 btw

Reply:
Then this needs a comment otherwise the next person is asking themselves possible the same thing as I did

Reply:
Added a comment on top of all the discord packages